docs/modules/schema-validation/proc-schema-validation.adoc

```diff
@@ -0,0 +1,83 @@
+// file included in the following:
+//
+// assembly-schema-validation-filter.adoc
+
+[id='proc-record-validation-{context}']
+= (Preview) Setting up the Record Validation filter
+
+
+.Prerequisites
+
+* An instance of Kroxylicious.
+For information on deploying Kroxylicious, see the link:{github}[samples and examples^].
```

Comment:
If we're reusing this procedure downstream, maybe we need a different approach to this prereq. 
It would just be an xref downstream as we have a deploying the proxy procedure. Perhaps we can add in a Deploying Kroxylicious section with the same id as downstream (proc-deploying-proxy-{context}) that links to the samples and examples?  Or we could add a conditional to keep the _For information_ line out of the downstream. I prefer to avoid conditionals.
I can look at this when we've merged this PR in, if you like.